korangar/src/graphics/passes/forward/entity.rs

```diff
@@ -28,16 +28,17 @@ const INITIAL_INSTRUCTION_SIZE: usize = 256;
 pub(crate) struct InstanceData {
     world: [[f32; 4]; 4],
     frame_part_transform: [[f32; 4]; 4],
+    action_rotation_matrix: [[f32; 2]; 2],
+    color: [f32; 4],
     texture_position: [f32; 2],
     texture_size: [f32; 2],
-    color: [f32; 4],
     extra_depth_offset: f32,
     depth_offset: f32,
-    angle: f32,
     curvature: f32,
     mirror: u32,
     texture_index: i32,
-    padding: [u32; 2],
+    padding: u32,
+    padding_2: [u32; 2],
```

Comment:
No need to use padding_2 here. Just make "padding" a [u32; 2].

Reply:
Was having some issue with depth offset and think that the shader was wrong, because of padding, but I was wrong